build.cmd

```diff
@@ -6,6 +6,12 @@ IF NOT EXIST packages\FAKE\tools\FAKE.exe  (
     .paket\paket.exe restore
 )
 
+cd .\src\FSharp.CloudAgent\
```

Comment:
this 5 lines can be replaced with just 
```
dotnet restore
```
it will do restore for all the projects in solution. Also please consider moving this to "fsx" file.

Reply:
Why fsx file? Ppl want to be able to just run from cmdline.

Reply:
in case people run "build.cmd" - it will run "build.fsx" and it will do "dotnet restore". Nothing is changed really. But doing "restore" is more a part of build process itself - not a bootstrap process in "build.cmd". Anyway this is just a suggestion.

Also currently if you skip "build.cmd" and run directly "FAKE.exe build.fsx" - there will be no "dotnet restore" and build will fail.

Reply:
That's standard behaviour re: FAKE directly. Indeed that could never happen because FAKE is itself pulled down by Paket (which is called by dotnet restore)